microsoft-edge/devtools-guide-chromium/whats-new/2022/02/devtools.md

```diff
@@ -0,0 +1,167 @@
+---
+title: What's new in DevTools (Microsoft Edge 98)
+description: 2021 year-in-review blog post.  Automate IE mode with Internet Explorer Driver.  Emulate forced-colors mode.  Tooltips for Activity icons in the Performance tool Event Log.  Shallow sizes in the Memory tool now shown as decimal values.  Network tool's Search textbox can now be resized.  And more.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 02/02/2022
+---
+# What's New in DevTools (Microsoft Edge 98)
+
+<!--
+todo:
+compare result against work items
+-->
+
+The following sections list the announcements from the Microsoft Edge DevTools team.  To try the latest features of DevTools and the Microsoft Edge DevTools extension for Visual Studio Code, read these announcements.  To stay up to date with the latest and greatest features in your developer tools, download the [Microsoft Edge preview channels](https://www.microsoftedgeinsider.com/download) and [follow the Microsoft Edge DevTools team on Twitter](https://twitter.com/EdgeDevTools).
+
+If you are on Windows, Linux, or macOS, consider using the Canary preview channel of Microsoft Edge as your default development browser.  The preview channels give you access to the latest features of Microsoft Edge DevTools.
+
+
+<!-- ====================================================================== -->
+## Catch up on the latest and greatest features in Microsoft Edge
+
+<!-- careful changing h2 wording: Welcome tool potentially links to it -->
+
+<!-- Title: Year-in-review: Microsoft Edge for developers -->
+<!-- Subtitle: Catch up on the latest in developer tooling from Microsoft Edge. -->
+
+In 2021, developer tooling for Microsoft Edge has taken a massive leap forward.  Visual Studio Code, DevTools, and the browser now provide several integrated workflows for JavaScript debugging, mirroring CSS changes, and screencasting from the browser.  Between Focus Mode, support for Visual Studio Code themes, and further customization options, the Microsoft Edge team is working to make the DevTools easier and simpler to use.
+
+Catch up on all of this news and more in our year-in-review blog post, [Looking back at Microsoft Edge for developers in 2021](https://blogs.windows.com/msedgedev/2022/01/19/looking-back-at-microsoft-edge-for-developers-in-2021/).
```

Comment:
Just wanted to check a layout issue. This second paragraph with the link and the catchup sentence is the important one for customers. When we do the layout in the welcome tool, I just want to make sure we don't bury the lede.

Reply:
@Reezaali @zoherghadyali - you know the later, Welcome tool context.  Is there some revision I should try making here?

Reply:
@JoshuaWeber - I think the layout in the Welcome tool you're referring to is populated by a copy of these comments, not what's displayed in this h2 section.  Maybe the Subtitle string should have phrases from 
"Catch up on all of this news and more in our year-in-review blog post, "Looking back at Microsoft Edge for developers in 2021""
not just "Catch up on the latest in developer tooling from Microsoft Edge."
<!-- Title: Year-in-review: Microsoft Edge for developers -->
<!-- Subtitle: Catch up on the latest in developer tooling from Microsoft Edge. -->

Reply:
@zoherghadyali - Closing this comment.  I think the Ask is to add "year-in-review blog post, looking back at Microsoft Edge for developers in 2021" to the Subtitle, "Catch up on the latest in developer tooling from Microsoft Edge."